lib/omnibus-ctl.rb

```diff
@@ -159,6 +159,10 @@ def load_files(path)
       end
     end
 
+    def load_file(filepath)
```

Comment:
Might want to use this same function in the load_files method above.  Otherwise :+1: